FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-beta2/c_sdk/directories.txt

```diff
@@ -0,0 +1,3 @@
++ standard
```

Comment:
Let's change folder to  FreeRTOS-Iot-Libraries-LTS-Beta2 ( instead of beta2 ) Archit is making similar change for Beta1